internal/cluster/cluster.go

```diff
@@ -484,6 +479,18 @@ func (m *Manager) SkipMonitoring(c *common.Cluster) bool {
 	return result
 }
 
+func (m *Manager) initMonitorQueryGenerator() {
+	if m.monitorQueryGenerator == nil {
+		noNeedToMonitorInStates := []string{
```

Comment:
maybe it should be consts?

Reply:
or some param inside monitor itself?

Reply:
Slices cannot be consts, or did you mean something else?

Reply:
I mean that noNeedToMonitorInStates should be more visible. It should be or part of monitor function itself or some global/consts?

Reply:
It is actually part of the query.  I can add function that just builds the query.  Is it OK?

Reply:
maybe initMonitorQueryGenerator should just get list of skippedStatuses?

Reply:
It can.  Is it better ?

Reply:
Maybe it can be declared in the beginning of a function and not part of if clause ?

Reply:
Getting a list isn't better.
The current implementation seems fine.